backend/entrypoint.sh

```diff
@@ -1,4 +1,5 @@
 #!bin/bash
-python manage.py makemigrations
+python manage.py makemigrations custom_models
```

Comment:
We should never have to change this script based on the model names. It should work no matter the migrations we add.

Reply:
Undo these changes.
See commit #https://github.com/code-for-canada/project-thundercat/pull/279/commits/41988f12f0f11475da97cf49dec74b9c9a21bfc5